bower.json

```diff
@@ -2,9 +2,11 @@
   "name": "paper-toast",
   "private": true,
   "dependencies": {
-    "core-media-query": "Polymer/core-media-query#master",
-    "core-overlay": "Polymer/core-overlay#master",
+    "polymer": "Polymer/polymer#0.8.0-rc.4",
     "font-roboto": "Polymer/font-roboto#master",
```

Comment:
We've replaced usage of this repo with `polymerelements/paper-styles`. You can pull in one of the typography imports to get the `roboto` font.